### PR TITLE
DL-4361 - remove allowlisting and pejorative terms

### PR DESCRIPTION
**Pull request by jeremystone**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -5,13 +5,13 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
...
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.8.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.1.0")
 
 addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.25")
```
Comment (paulanderson1234) [inline project/plugins.sbt:14]: Can sbt-plugin be upgraded to 2.7.5 as per catalogue?

File: project/plugins.sbt [hunk 2]
```diff
@@ -5,13 +5,13 @@ resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
...
 resolvers += Resolver.typesafeRepo("releases")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.10.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "2.13.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.1.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "2.2.0")
 
-addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.8.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.13.0")
```
Comment (paulanderson1234) [inline project/plugins.sbt]: We may be able to do away with this as it is imported by jenkins automatically:https://confluence.tools.tax.service.gov.uk/pages/viewpage.action?spaceKey=TEC&postingDay=2021%2F1%2F18&title=Removing+the+sbt-artifactory+plugin

Comment (jeremystone) [reply project/plugins.sbt]: 👍

File: project/plugins.sbt (hunk 1 above)
Comment (jeremystone) [reply project/plugins.sbt:14]: Unfortunately this pulls in play 2.7 and breaks a few things.

**Pull request merged**